src/20240617_AaveV3Avalanche_UpdatePoRExecutorV3Robot/UpdatePoRExecutorV3Robot.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Update PoR Executor V3 Robot"
+author: "BGD Labs (@bgdlabs)"
+discussions: ""
+---
+
+## Simple Summary
+
+Proposal to give new Proof of Reserve Executor V3 risk admin role on Avalanche and register the new robot.
+
+## Motivation
+
+With the release of Aave V3.1, it is no longer necessary to set the asset's LTV to zero before freezing during the execution of an emergency action on a Proof of Reserve alert. Moreover setting LTV to zero will break the setting of LTV back to normal value on unfreeze.
+
+The robot for the updated version of [Proof of Reserve Executor](https://snowscan.xyz/address/0xb94e515615c244ab25f7a6e592e3cb7ee31e99f4) is registered in this proposal and the existing one is being canceled.
+
+The new version of the Proof of Reserve Executor is granted the risk admin role.
+
+## Specification
```

Comment:
Can you add here a table with the new addresses involved? The new PoR Executor, new robot

Reply:
Also, I think important in this case (especially to facilitate transparency/review by others) to describe how the 2 actions are planned to be executed sequentially